README.md

```diff
@@ -0,0 +1,1251 @@
+    name: Vega
+    time: ""
+    upgraded_client_state: null
+  title: Vega
+deposit_end_time: ""
+final_tally_result:
+  abstain: "0"
+  "no": "0"
+  no_with_veto: "0"
+  "yes": "0"
+proposal_id: "54"
+status: PROPOSAL_STATUS_PASSED
+submit_time: ""
+total_deposit:
+- amount: "100"
+  denom: uatom
+voting_end_time: ""
+voting_start_time: ""
+```
+
+After `PROPOSAL_STATUS_PASSED`, wait till the upgrade height is reached and comovisor will auto-download the new binary fitting your platform and apply upgrade. Please note, the link of binary for GOOS=darwin GOARCH=arm64(for mac M1 users) is from this repo, not from our releases (presently we do not provide binary for this platform in our release).
+
+## Upgrade result
+### Method I:
+When the upgrade height is reached, you can find info. in the log:  `ERR UPGRADE "Vega" NEEDED at height: 7368587: upgrade to Vega` and `applying upgrade "Vega" at height:7368587`. Then the chain will progress to produce blocks after both nodes upgrade.
+### Method II:
+When the upgrade height is reached, you can find info. in the log: `INF applying upgrade "Vega" at height: 7368587`. Then chain will progress to produce blocks after both nodes upgrade.
+
+## Repeating the test
+
+If you want to try running the test again, or if you made a mistake while running it the first time, make sure to do the following steps:
+
+1. `cd vega-test`,`gaiad unsafe-reset-all --home data/test/val1` and `gaiad unsafe-reset-all --home data/test/val2`
+2. If you ran the upgrade successfully with cosmovisor `auto-download` enabled, make sure that you remove the downloaded binary from `data/test/val1/cosmovisor/upgrades/Vega` and `data/test/val2/cosmovisor/upgrades/Vega`. Also remove the symbolic link with `rm data/test/val1/cosmovisor/current` and `rm data/test/val2/cosmovisor/current`.
+Now you should be able to start the two binaries again and perform the upgrade from the start.
+
+## Further info: test new modules
+Now you can explore the functions of new modules in gaia.
+For authz module, you can refer https://github.com/cosmos/sdk-tutorials/pull/786/files for further testing.
+
+## Reference
+
+[cosmovisor quick start](https://github.com/cosmos/cosmos-sdk/tree/master/cosmovisor)
+
+[changelog of cosmos-sdk v0.43.0](https://github.com/cosmos/cosmos-sdk/blob/v0.43.0/CHANGELOG.md#v0430---2021-08-10)
+
+[cosmos/ibc-go v1.0.0](https://github.com/cosmos/ibc-go/tree/v1.0.0)
+
+[Gravity DEX Upgrade Simulation Test](https://github.com/b-harvest/gravity-dex-upgrade-test/blob/kogisin/v5.0.5-upgrade-simulation/v5.0.5/README.md)
+
+[cosmovisor](https://github.com/cosmos/cosmos-sdk/tree/master/cosmovisor)
+## changes in genesis file
```

Comment:
```suggestion
When the upgrade block height is reached, you can find the following information in the log: `ERR UPGRADE "Vega" NEEDED at height: 7368587: upgrade to Vega` and `applying upgrade "Vega" at height:7368587`. The chain will continue to produce blocks after both nodes have completed the upgrade.
### Method II:
When the upgrade block height is reached, you can find the following information in the log: `INF applying upgrade "Vega" at height: 7368587`. The chain will continue to produce blocks after both nodes have completed the upgrade.

## Repeating the test

If you want to try running the test again, or if you made a mistake while running it the first time, make sure to do the following steps:

1. `cd vega-test`,`gaiad unsafe-reset-all --home data/test/val1` and `gaiad unsafe-reset-all --home data/test/val2`
2. If you ran the upgrade successfully with the Cosmovisor `auto-download` flag enabled, make sure that you remove the downloaded binary from `data/test/val1/cosmovisor/upgrades/Vega` and `data/test/val2/cosmovisor/upgrades/Vega`. Also remove the symbolic link with `rm data/test/val1/cosmovisor/current` and `rm data/test/val2/cosmovisor/current`.
Now you should be able to start the two binaries again and perform the upgrade from the start.

## Further info: test new modules
Now you can explore some of the functions of the new modules in Gaia.
For the AuthZ module, please refer to https://docs.cosmos.network/master/modules/authz/

## Reference

[Cosmovisor quick start](https://github.com/cosmos/cosmos-sdk/tree/master/cosmovisor)

[Changelog of cosmos-sdk v0.43.0](https://github.com/cosmos/cosmos-sdk/blob/v0.43.0/CHANGELOG.md#v0430---2021-08-10)

[cosmos/ibc-go v1.0.0](https://github.com/cosmos/ibc-go/tree/v1.0.0)

[Gravity DEX Upgrade Simulation Test](https://github.com/b-harvest/gravity-dex-upgrade-test/blob/kogisin/v5.0.5-upgrade-simulation/v5.0.5/README.md)

[Cosmovisor](https://github.com/cosmos/cosmos-sdk/tree/master/cosmovisor)
## Changes in the genesis file
```

Regarding AuthZ module, maybe it's better to refer to https://docs.cosmos.network/master/modules/authz/ ? What do you think?